lib/utils/dependency-manager.js

```diff
@@ -21,18 +21,23 @@ const dfOps = require('./directory-file-ops.js')
 const npm = /^win/.test(process.platform) ? "npm.cmd" : "npm" 
 const yarn = /^win/.test(process.platform) ? "yarn.cmd" : "yarn" 
 
-function npmInstall(projectPath, callback){
-    let packageJsonFilePath = path.normalize(path.join(projectPath, 'package.json'))
+function installDependencies(projectInfo, callback){
+    let packageJsonFilePath = path.normalize(path.join(projectInfo.ProjectPath, 'package.json'))
     if(fs.existsSync(packageJsonFilePath)){
         console.log()
-        console.log('Executing ' + npm + ' install...')
-        let childProcess = spawn(npm, ['install'], {cwd: projectPath, env: process.env, stdio: 'inherit'})
+        let command = npm
+        if(projectInfo.Framework=='react' || projectInfo.Framework == 'react-native'){
+            command = yarn
```

Comment:
Wouldn't the CLI error out if yarn is not installed?

Reply:
good catch, I added the check.